kotlin-react-router-dom/src/main/kotlin/react/router/dom/Route.kt

```diff
@@ -3,23 +3,22 @@
 
 package react.router.dom
 
-import react.ComponentClass
-import react.ComponentType
-import react.Props
-import react.ReactElement
+import react.*
 
-external interface RouteProps : Props {
+external interface RouteProps : PropsWithChildren {
```

Comment:
No `children` detected

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e12cc2aedf5ee2ff668a9b295231d0c32af35285/types/react-router/index.d.ts#L86

Reply:
I am agree with @aerialist7. Children is most modern API and it is recommended since hooks promotion

Reply:
https://reactrouter.com/web/api/Route/route-render-methods
> The recommended method of rendering something with a <Route> is to use children elements